### ISSUE TITLE
Theming not saved

### ISSUE THREAD

**Issue by Rouzax** [nextcloud/server#401]
### Steps to reproduce
1. Open the Theming options
2. Change settings like color or logo (changes are immediately visible)
3. Refresh page
4. Settings are still there but actual logo or color are reverted back to default
![image](https://cloud.githubusercontent.com/assets/4103090/16841652/85579e0a-49da-11e6-84b1-f7fe0dafb6ea.png)

After refresh
![image](https://cloud.githubusercontent.com/assets/4103090/16841611/4c7c4c02-49da-11e6-9000-cea0d5d94473.png)

### Server configuration
**Operating system**:

**Web server:**

**Database:**
MySQL

**PHP version:**
PHP7

**Nextcloud version:** (see Nextcloud admin page)
Nextcloud 9.0.52 (stable)

**Updated from an older Nextcloud/ownCloud or fresh install:**
Upgraded from OC 9

**Where did you install Nextcloud from:**
zip

**Signing status (Nextcloud 9.0 and above):**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>default apps</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

**The content of config/config.php:**
<details>
<summary>Not available</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

</details>

### Client configuration
**Browser:**
Chrome Version 51.0.2704.103 m (64-bit)
 
**Operating system:**
Windows 8.1

#### Web server error log
<details>
<summary>Web server error log</summary>
```
Nothing in the log regarding this issue
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>
```
Insert your Nextcloud log here
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by LukasReschke** [comment]
Please provide a bit more information such as the network log and the operating system and web server. That said I suspect this is a duplicate of https://github.com/nextcloud/server/issues/359 / https://github.com/nextcloud/server/issues/315 and you may be able to use https://github.com/nextcloud/server/pull/370

**Comment by Rouzax** [comment]
@LukasReschke What do you mean with the network log?
Apache Version	2.4.18
MySQL Version	5.6.30
Architecture	x86_64
Operating System	linux

**Comment by LukasReschke** [comment]
Please test https://github.com/nextcloud/server/pull/370 as linked above.

> @LukasReschke What do you mean with the network log?

Browser's network console. There an request to the theming CSS file is sent and it likely fails.

**Comment by sndrr** [comment]
AFAIK I cannot reproduce this behavior with the latest version from [master](https://github.com/nextcloud/server/commit/3b386af295fdd5ea986edd4447637043f69b4c38)

**Comment by MorrisJobke** [comment]
> Please test #370 as linked above.

Will be fixed with the next release - 9.0.54.
